packages/appstore/src/constants/trading-hub-content.tsx

```diff
@@ -13,39 +14,67 @@ type TTradingHubContents = Record<'step1' | 'step2' | 'step3' | 'step4' | 'step5
 
 export const trading_hub_contents: TTradingHubContents = {
     step1: {
-        component: 'Step1',
+        component: <StaticDashboard />,
```

Comment:
the logo and title  of financial and derived accounts in CFD part is different from design, also non of the elements should be clickable

test link:
![image](https://user-images.githubusercontent.com/82078941/192279916-6aa1d5fe-0052-4099-a19b-237964fd874a.png)

design:
![image](https://user-images.githubusercontent.com/82078941/192280016-c6fb0e64-fc6b-4877-9e9d-655615c7853e.png)